packages/wonder-blocks-banner/src/components/banner.js

```diff
@@ -88,20 +94,38 @@ type Props = {|
      * If present, dismiss button is on right side. If not, no button appears
      */
     onDismiss?: ?() => void,
+
+    /**
+     * The accessible label for the dismiss button.
+     * Please pass in a translated string.
+     */
+    dismissAriaLabel: string,
 |};
 
-const colorForKind = (kind: BannerKind) => {
+const valuesForKind = (kind: BannerKind): BannerValues => {
     switch (kind) {
-        case "info":
-            return Color.blue;
         case "success":
-            return Color.green;
+            return {
+                color: Color.green,
+                role: "status",
+            };
         case "warning":
-            return Color.gold;
+            return {
+                color: Color.gold,
+                role: "alert",
+                ariaLive: "polite",
+            };
         case "critical":
-            return Color.red;
+            return {
+                color: Color.red,
+                role: "alert",
+                ariaLive: "assertive",
```

Comment:
**nit:** I think `assertive` is defined by default with the `alert` role, so there might not be need to add this key here.

For reference: https://www.w3.org/TR/wai-aria-1.1/#alert

> Elements with the role alert have an implicit [aria-live](https://www.w3.org/TR/wai-aria-1.1/#aria-live) value of assertive, and an implicit [aria-atomic](https://www.w3.org/TR/wai-aria-1.1/#aria-atomic) value of true.